gm4_tnt_landmines/data/tnt_landmines/functions/fix_skull.mcfunction

```diff
@@ -0,0 +1,11 @@
+execute if entity @s[nbt={Item:{tag:{SkullOwner:{Id:"da309950-dcbc-43f2-880e-3820cffea8ca",Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQ0MjA4NDgsInByb2ZpbGVJZCI6IjkxZjA0ZmU5MGYzNjQzYjU4ZjIwZTMzNzVmODZkMzllIiwicHJvZmlsZU5hbWUiOiJTdG9ybVN0b3JteSIsInNpZ25hdHVyZVJlcXVpcmVkIjp0cnVlLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZGI1Mzc5ZjE2NWE1Y2M3YmI4YzM4MmZlOTJlMjRlODYwN2NiODU1Y2FjNmU4M2Q4NjJjNjBlZmNlZTA5MTE3NSJ9fX0="}]}}}}}] run data merge entity @s {Item:{tag:{gm4_tnt_landmines:{item:1,block:"dirt"},display:{Name:'{"translate":"%1$s%3427655$s","with":["Landmine",{"translate":"item.gm4.tnt_landmine"}]}',Lore:['{"translate":"%1$s%3427655$s","with":["Drop to use",{"translate":"item.gm4.tnt_landmine.use"}]}','[{"translate":"%1$s%3427655$s","with":["Disguised as ",{"translate":"item.gm4.tnt_landmine.disguised"}]},{"translate":"block.minecraft.dirt"}]']}}}}
+execute if entity @s[nbt={Item:{tag:{SkullOwner:{Id:"ec0d8396-3a18-4298-842c-de978a5ee5c6",Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQxODg2MTMsInByb2ZpbGVJZCI6IjNmYzdmZGY5Mzk2MzRjNDE5MTE5OWJhM2Y3Y2MzZmVkIiwicHJvZmlsZU5hbWUiOiJZZWxlaGEiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzNjNjk4ZTE5MDllMWI4MzMxYmVhOGQ1YzU3N2NkMGM2M2I4YWVjMDFmYjg4ZjY0OWUwMGYwZmU5NjY1NDdjNWIifX19"}]}}}}}] run data merge entity @s {Item:{tag:{gm4_tnt_landmines:{item:1,block:"stone"},display:{Name:'{"translate":"%1$s%3427655$s","with":["Landmine",{"translate":"item.gm4.tnt_landmine"}]}',Lore:['{"translate":"%1$s%3427655$s","with":["Drop to use",{"translate":"item.gm4.tnt_landmine.use"}]}','[{"translate":"%1$s%3427655$s","with":["Disguised as ",{"translate":"item.gm4.tnt_landmine.disguised"}]},{"translate":"block.minecraft.stone"}]']}}}}
+execute if entity @s[nbt={Item:{tag:{SkullOwner:{Id:"2bedd9c9-b534-4c25-992a-2c5acfaac23f",Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NTQ0NzQyMjMxMzEsInByb2ZpbGVJZCI6ImIwZDczMmZlMDBmNzQwN2U5ZTdmNzQ2MzAxY2Q5OGNhIiwicHJvZmlsZU5hbWUiOiJPUHBscyIsInNpZ25hdHVyZVJlcXVpcmVkIjp0cnVlLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNDJhOGE5NDk5MjE5NTk1NzEyYjI1ODdlZmViMzViMWY2NTU2ZGExMmIzZWU4ODI5NGUwMzk0MDM0YmZjMjhlOCJ9fX0="}]}}}}}] run data merge entity @s {Item:{tag:{gm4_tnt_landmines:{item:1,block:"stone_bricks"},display:{Name:'{"translate":"%1$s%3427655$s","with":["Landmine",{"translate":"item.gm4.tnt_landmine"}]}',Lore:['{"translate":"%1$s%3427655$s","with":["Drop to use",{"translate":"item.gm4.tnt_landmine.use"}]}','[{"translate":"%1$s%3427655$s","with":["Disguised as ",{"translate":"item.gm4.tnt_landmine.disguised"}]},{"translate":"block.minecraft.stone_bricks"}]']}}}}
```

Comment:
In some languages, the block name may come before or in the middle of the text. So you should really use `%s` here, even though the text component gets quite complicated.
```
'[{"translate":"%1$s%3427655$s","with":[["Disguised as ",{"translate":"block.minecraft.stone"}],{"translate":"item.gm4.tnt_landmine.disguised","with":[{"translate":"block.minecraft.stone"}]}]}]'
```